libs/metagen/src/tests/fixtures.rs

```diff
@@ -0,0 +1,97 @@
+use crate::interlude::*;
+use common::typegraph::*;
+
+pub async fn test_typegraph_1() -> anyhow::Result<Typegraph> {
+    let out = tokio::process::Command::new("cargo")
+        .args(
+            // "run -p meta-cli -- serialize -f tests/tg.ts"
+            "run -p meta-cli -- serialize -f ../../examples/typegraphs/reduce.py"
+                .split(' ')
+                .collect::<Vec<_>>(),
+        )
+        .kill_on_drop(true)
+        .output()
+        .await?;
+    let mut tg: Vec<Typegraph> = serde_json::from_slice(&out.stdout)
+        .with_context(|| format!("error deserializing typegraph: {out:?}"))?;
+    Ok(tg.pop().unwrap())
+}
+
+#[allow(unused)]
+pub fn test_typegraph_2() -> Typegraph {
+    Typegraph {
+        id: "https://metatype.dev/specs/0.0.3.json".into(),
+        path: None,
+        policies: vec![],
+        deps: vec![],
+        meta: TypeMeta {
+            ..Default::default()
+        },
+        runtimes: vec![common::typegraph::runtimes::TGRuntime::Unknown(
+            common::typegraph::runtimes::UnknownRuntime {
+                name: "wasm".into(),
+                data: Default::default(),
+            },
```

Comment:
Is the use of this runtime part of your design or is it just for testing?

Reply:
Was for testing.